README.md

```diff
@@ -1 +1,36 @@
-# HELLOMICROVERSE
\ No newline at end of file
+# Set-up-a-Hello-Microverse-project
```

Comment:
- [ ] `Set-up-a-Hello-Microverse-project`  is supposed to be project title. As you can guess it is not recommended to write titles with Dashes. Try to keep titles simple, eg: `Hello Microverse`